src/LifeCycle.hs

```diff
@@ -137,9 +150,24 @@ userCapabilities = \case
 
 -- * Idea Capabilities
 
-
-ideaCapabilities :: AUID User -> Role -> Idea -> Phase -> [Capability]
-ideaCapabilities = phaseCap
+ideaCapabilities :: AUID User -> Role -> Idea -> Phase -> [Clickable Capability]
+ideaCapabilities u r i p =
+    let activeCaps = nub $ ideaCapabilitiesInPhase u r i p
+        allCaps    = nub $ ideaCapabilitiesInAllPhases u r i
+    in (Clickable <$> activeCaps) <> (GrayedOut <$> (allCaps \\ activeCaps))
+
+-- ASSUMPTION: The list of a phase should be part of the AllPhase list.
```

Comment:
Can you rephrase this assumption?  My best guess is you mean this (but I suspect I got it wrong):

```haskell
-- The union of the dea caps of all phases.
```